README.md

```diff
@@ -16,9 +16,9 @@ This is designed to handle failure gracefully. If processing of a survey fails,
 
 ActionKit tables can be in a separate schema, but currently must be accessible within the same database connection.
 
-## Settings
+## settings.py
 
-Settings are fetched directly from AWS Secrets Manager using the secret name `ak-survey-results`. 
```

Comment:
quick question: did this change get reverted on purpose or by accident? When we were pairing, we were working off of the settings.py file, but I know these changes involve migrating to secrets manager